src/iidm/Network.cpp

```diff
@@ -43,6 +49,15 @@ namespace powsybl {
 
 namespace iidm {
 
+static const converter::Parameter ANONYMISED_PARAMETER(converter::xml::ANONYMISED, converter::Parameter::ParameterType::BOOLEAN, "Anonymise exported network", false);
+static const converter::Parameter INDENT_PARAMETER(converter::xml::INDENT, converter::Parameter::ParameterType::BOOLEAN, "Indent export output file", true);
+static const converter::Parameter EXTENSIONS_LIST_PARAMETER(converter::xml::EXTENSIONS_LIST, converter::Parameter::ParameterType::STRING_LIST, "The list of exported extensions", std::set<std::string>());
+static const converter::Parameter ONLY_MAIN_CC_PARAMETER(converter::xml::ONLY_MAIN_CC, converter::Parameter::ParameterType::BOOLEAN, "Export only main CC", false);
+static const converter::Parameter THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND_PARAMETER = std::move(converter::Parameter(converter::xml::THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND, converter::Parameter::ParameterType::BOOLEAN, "Throw exception if extension not found", false).addAdditionalNames({"throwExceptionIfExtensionNotFound"}));
+static const converter::Parameter TOPOLOGY_LEVEL_PARAMETER(converter::xml::TOPOLOGY_LEVEL, converter::Parameter::ParameterType::STRING, "Export network in this topology level", std::string("NODE_BREAKER"));
+static const converter::Parameter VERSION_PARAMETER(converter::xml::VERSION, converter::Parameter::ParameterType::STRING, "IIDM-XML version in which files will be generated", converter::xml::IidmXmlVersion::CURRENT_IIDM_XML_VERSION().toString("."));
+static const converter::Parameter WITH_BRANCH_STATE_VARIABLES_PARAMETER(converter::xml::WITH_BRANCH_STATE_VARIABLES, converter::Parameter::ParameterType::BOOLEAN, "Export network with branch state variables", true);
```

Comment:
Maybe you should move these options in ImportOptions and ExportOptions.

A single constructor could be OK, if it's you use a sting parameter for the default value.